core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java

```diff
@@ -52,7 +56,7 @@ public long getServiceVersion() {
 
   @Override
   public List<WorkerInfo> getWorkerInfoList() throws AlluxioTException {
-    return RpcUtils.call(new RpcCallable<List<WorkerInfo>>() {
+    return RpcUtils.call(LOG, new RpcCallable<List<WorkerInfo>>() {
```

Comment:
Everything which is not automatic/periodic should eventually use `callAndLog`, currently I am only targeting the 3 most used service handlers and based on the usefulness of this approach, we can  expand the use to other handlers.